.github/javascript-tests.yml

```diff
@@ -0,0 +1,26 @@
+name: Run Javascript Tests
```

Comment:
Try putting this in a folder called `workflows` , so should look like `.github/workflows/javascript-test.yml`. Hopefully this will show the tests in Github Actions